projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -1000,6 +1000,14 @@ export interface FeatureTogglesInterface {
    */
   enableSecurePasswordValidation?: boolean;
 
+  /**
+   * When enabled, the `ConfiguratorAttributeHeaderComponent` component displays
+   * a `ConfiguratorShowOptionsComponent` component underneath the attribute name.
+   * The `ConfiguratorShowOptionsComponent` component allows to load the domain values
+   * on demand by clicking on `Show Options` button.
+   */
+  readDomainValuesOnDemand?: boolean;
```

Comment:
Shouldn't we call it 
```suggestion
  enableReadDomainValuesOnDemand?: boolean;
```